src/applications/check-in/pre-check-in/README.md

```diff
@@ -97,7 +97,9 @@ Though we have the HOC, its now considered best practice to query redux using th
   - when to sunset: once we have successfully tested this feature in production with users
 - `check_in_experience_lorota_security_updates_enabled` : Enables or disables DOB log in instead of last 4 of SSN
   - when to sunset: once we have successfully tested this feature in production with users and the backend has full switched over
-  - 
+- `check_in_experience_edit_messaging_enabled` : Enables or disables edit messaging.
+  - when to sunset: once we have successfully tested this feature in production with users and the backend has full switched over
```

Comment:
There's a minor typo in the description here. "backend has full switched over".

Reply:
thanks @dcgoodwin2112 got it fixed on both readme files